colossalai/shardformer/layer/__init__.py

```diff
@@ -16,6 +23,7 @@
     "Linear1D_Row",
     "GPT2FusedLinearConv1D_Col",
     "GPT2FusedLinearConv1D_Row",
+    "GPT2FusedLinearConv1D_Col",
```

Comment:
This line is the same as line 24.

Reply:
> This line is the same as line 24.


Remove communication in https://github.com/hpcaitech/ColossalAI/pull/6150/commits/7fb23a57d83cd9f876d9024079ab922f5e771852;